SPECS/erlang/erlang.spec

```diff
@@ -19,52 +24,59 @@ erlang programming language
 
 %build
 export ERL_TOP=`pwd`
-./otp_build autoconf
-sh configure --disable-hipe --prefix=%{_prefix}
-
+./configure
```

Comment:
Can we call the `%configure` macro here? It will set our hardened build flags and should fix the `/usr/local/bin` problem Pawel pointed out.

Reply:
After updating the spec file to use this `%configure`, I ran both an AMD64 and ARM64 buddy build, with successful results, indicating that in fact was the issue causing the use of the default install location of `/usr/local/bin` and `/usr/local/lib`. The define tags have been removed accordingly and files listed in the `%files` section of the spec have been updated to use `_bindir` and `_libdir` respectively.